plugins/chrome-ux-report/src/components/ChromeUXReportChart.tsx

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { useApi, Progress } from '@backstage/core';
+import { chromeuxReportApiRef } from '../api';
+import { useAsync } from 'react-use';
+import Alert from '@material-ui/lab/Alert';
+import { Chart } from 'react-google-charts';
+import { getRoundOfMetric } from '../utils';
+import { ChromeUXReportPeriod } from './ChromeUXReportPeriod';
+
+export const ChromeUXReportChart = ({ origin }: { origin: string }) => {
+  const chromeUXReportApi = useApi(chromeuxReportApiRef);
+
+  const { value, loading, error } = useAsync(async (): Promise<any> => {
+    const response = await chromeUXReportApi.getChromeUXMetrics(origin);
+    return response.metrics;
+  }, []);
+
+  if (loading) {
+    return <Progress />;
+  } else if (error) {
+    return <Alert severity="error">{error.message}</Alert>;
```

Comment:
There's a convenient `<ResponseErrorPanel error={error} />` in core that you could use instead if you handle your service responses in your client like this https://github.com/backstage/backstage/blob/master/packages/catalog-client/src/CatalogClient.ts#L230